mentors/scala/b.json

```diff
@@ -1 +1,10 @@
-[]
+[
+{
```

Comment:
```suggestion
  {
```
indent two spaces.  Lines that follow `{` should also be indented two spaces in from `[`.